src/pages/home/components/nav-bar.module.scss

```diff
@@ -0,0 +1,61 @@
+.navBar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: sticky;
+  top: 0;
+  width: 100%;
+  gap: 10%;
+  background-color: white;
+  height: 60px;
+  border-bottom: solid;
+  border-width: 1px;
+  border-color: #cecece;
+  font-size: large;
+
+  & form {
+    width: 300px;
+  }
+
+  & input {
+    width: 100%;
+    padding-left: 40px;
+    border-radius: 10px;
+    background-color: #ededed;
+    border: none;
+    height: 40px;
+  }
+
+  & input:focus{
+    outline: none !important;
```

Comment:
does it just work with !important ? Using it is usually something to we would want to avoid.